wallet/res/layout/activity_staking.xml

```diff
@@ -23,6 +23,12 @@
     android:background="@color/background_primary"
     tools:context="de.schildbach.wallet.ui.staking.StakingActivity">
 
+    <ProgressBar
+        android:id="@+id/progress_bar"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_gravity="center" />
```

Comment:
Shows up on the `StakingActivity` before navigation is set in case we need to fetch `GetFunds` first.